rstr/rstr_base.py

```diff
@@ -157,3 +160,7 @@ def __init__(self, _random=random, **alphabets):
         super(Rstr, self).__init__(_random=_random, **alphabets)
 
 default_instance = Rstr()
+
+
+class SameCharacterError(Exception):
+    pass
```

Comment:
Newline here please. :slightly_smiling_face:

Reply:
Inherit `SameCharacterError` from `ValueError` because it's a problem with the values passed.

Reply:
I fixed them in a new commit.